### DISCUSSION TITLE
Issue with Rolling Restart in OpenSearch Operator 2.7.0 and OpenSearch 2.17.1.0

### DISCUSSION THREAD

**Discussion by rameshar16** [Opster/opensearch-k8s-operator#916] [General]
Hi OpenSearch Community,

I am currently using OpenSearch Operator version 2.7.0 and OpenSearch version 2.17.1.0. I have noticed that rolling restarts are not occurring for any configuration or security changes.

Is anyone else experiencing similar behavior? Any insights or suggestions on how to resolve this issue would be greatly appreciated.

Thanks and Regards,

**Comment by swoehrl-mw** [comment]
Hi @rameshar16 Could you be more specific what you mean by "configuration or security changes"? Can you provide a concrete cluster manifest with changes you are making that are not triggering a restart to reproduce?

**Reply by rameshar16** [reply]
Thank you @swoehrl-mw  for the quick. I am able to found the issue. There is a config issue.
